test/fixtures/linter/rules/NoGlobals/NoExportedLibValues_Negative.js

```diff
@@ -0,0 +1,20 @@
+sap.ui.define(
+	[
+		"sap/ui/unified/library",
+		"sap/ui/core/library",
+		// This ones are correct- elements, imported as modules
+		"sap/ui/unified/DateTypeRange",
+		"sap/ui/unified/CalendarDayType",
```

Comment:
This is actually a pseudo module (see finding in test snapshot), and it should rather be accessed via the library module.

Reply:
Yep, and that's why I added it. With the prevous miss, actually the pseudo modules were also reported as non exported values. Let's leave this as a safeguard.